docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -34,17 +34,15 @@ This topic describes how to:
 
 ## Before you begin
 
-* Ensure that you have basic familiarity with instrumenting applications with
-  OpenTelemetry.
-* Have a set of OpenTelemetry applications ready to push telemetry data to
-  {{< param "PRODUCT_NAME" >}}.
+* Ensure that you have basic familiarity with instrumenting applications with OpenTelemetry.
+* Have a set of OpenTelemetry applications ready to push telemetry data to {{< param "PRODUCT_NAME" >}}.
 * Identify where {{< param "PRODUCT_NAME" >}} will write received telemetry data.
 * Be familiar with the concept of [Components][] in {{< param "PRODUCT_NAME" >}}.
 * Complete the [Collect open telemetry data][] getting started guide. You will pick up from where that guide ended.
 
 ## The pipeline
 
-You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the previous getting started guide:
+You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the previous Getting Started guide.
```

Comment:
```suggestion
You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the [OpenTelemetry to Grafana stack][] Getting Started guide.
```
I think this should be a link to the guide that we are referring to.

Reply:
This should probably be `[Collect open telemetry data][]` instead :-)

Fixed in the next commit.